From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 6 Oct 2023 10:46:21 -0500
Subject: [PATCH] Downstream default settings

Changes:
- Default to system theme
- Toast notifications don't integrate as well as native ones
---
 src/librssguard/definitions/definitions.h      | 2 +-
 src/librssguard/miscellaneous/settingskeys.cpp | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/librssguard/definitions/definitions.h b/src/librssguard/definitions/definitions.h
index e0a8c2a22..f87ed4b13 100644
--- a/src/librssguard/definitions/definitions.h
+++ b/src/librssguard/definitions/definitions.h
@@ -250,7 +250,7 @@
 #define APP_NO_THEME                ""
 
 // Use OS default style everywhere.
-#define APP_STYLE_DEFAULT "Fusion"
+#define APP_STYLE_DEFAULT APP_NO_THEME
 
 // Column indexes for the model.
 #define MSG_MDL_ID_INDEX          0
diff --git a/src/librssguard/miscellaneous/settingskeys.cpp b/src/librssguard/miscellaneous/settingskeys.cpp
index a3c873129..e4b890bb1 100644
--- a/src/librssguard/miscellaneous/settingskeys.cpp
+++ b/src/librssguard/miscellaneous/settingskeys.cpp
@@ -389,7 +389,7 @@ DKEY GUI::EnableNotifications = QSL("enable_notifications");
 DVALUE(bool) GUI::EnableNotificationsDef = true;
 
 DKEY GUI::UseToastNotifications = QSL("use_toast_notifications");
-DVALUE(bool) GUI::UseToastNotificationsDef = true;
+DVALUE(bool) GUI::UseToastNotificationsDef = false;
 
 DKEY GUI::ToastNotificationsPosition = QSL("toast_notifications_position");
 DVALUE(int)
@@ -440,7 +440,7 @@ DKEY GUI::ColoredBusyTrayIcon = QSL("colored_busy_tray_icon");
 DVALUE(bool) GUI::ColoredBusyTrayIconDef = false;
 
 DKEY GUI::ForcedSkinColors = QSL("forced_skin_colors");
-DVALUE(bool) GUI::ForcedSkinColorsDef = true;
+DVALUE(bool) GUI::ForcedSkinColorsDef = false;
 
 DKEY GUI::UnreadNumbersInTrayIcon = QSL("show_unread_numbers_in_tray_icon");
 DVALUE(bool) GUI::UnreadNumbersInTrayIconDef = true;
